h5p-bildetema/package.json

```diff
@@ -21,29 +21,29 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@headlessui/react": "^1.7.18",
-    "@tanstack/react-query": "^4.35.3",
+    "@headlessui/react": "^1.7.19",
+    "@tanstack/react-query": "^5.34.1",
```

Comment:
I think this update is the one which needs actual testing the most. Should be enough to try open the page and see that data loads succesfully.